components/esp32_ble_tracker.rst

```diff
@@ -86,6 +87,13 @@ Configuration variables:
     asked to start a scan (with start_scan action). Defaults to ``true``.
 
 - **id** (*Optional*, :ref:`config-id`): Manually specify the ID for this ESP32 BLE Hub.
+- **max_connections** (*Optional*, int): The maximum BLE connections slots to use.
+  Each configured slot consumes ~1KB of RAM. It is recommended not to exceed ``5``
+  connections slots to avoid memory issues. Defaults to ``3``.
```

Comment:
```suggestion
- **max_connections** (*Optional*, int): The maximum number of BLE connection slots to use.
  Each configured slot consumes ~1KB of RAM. It is recommended not to exceed ``5``
  connection slots to avoid memory issues. Defaults to ``3``.
```